src/shared/install.c

```diff
@@ -743,8 +743,14 @@ static int remove_marked_symlinks_fd(
                                         continue;
                                 }
 
+                                r = path_extract_filename(dest, &fname);
+                                if (r < 0) {
+                                        log_error_errno(r, "Failed to extract filename from path %s: %m", dest);
```

Comment:
in systemd we have this rule that functions should either log about all their errors with a log level above LOG_DEBUG or about none (in the latter case the caller can do that instead). For each function one should decide which kind it is. So far this function here did not log about any error above LOG_DEBUG, hence please stick to this. i.e. downgrade this from log_error_errno() to log_debug_errno()